### PR TITLE
Addresses #1066: Implemented a button to Show and Hide Charts

### PR DESCRIPTION
**Pull request by praveenojha33**
Addresses #1066 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `master` branch.
- [x] I have added necessary documentation (if appropriate)
- [x] Added Surge preview link

#### Changes proposed in this pull request:
I have implemented a a button to show and hide charts. I submit later patches to display it on a light box.
Surge Link: https://pr-1081-fossasia-susper.surge.sh
image
image

### CODE REVIEW

**Comment by praveenojha33** [comment]
@simsausaurabh Please review, I will work on displaying it on light box in further patches.

File: src/app/statsbox/statsbox.component.css [hunk 1]
```diff
@@ -8,6 +8,22 @@
   width: 454px;
   margin-top: 10px;
 }
+#toggle-chart-button{
+    margin-top: 1vh;
+    margin-bottom: 1vh;
+    background-color: white;
+    border-color: lightgray;
+    border-radius: 0;
```
Comment (simsausaurabh) [inline src/app/statsbox/statsbox.component.css]: It can be removed.

File: src/app/statsbox/statsbox.component.ts [hunk 2]
```diff
@@ -16,6 +16,8 @@ export class StatsboxComponent implements OnInit {
   selectedelement: number = -1;
   querychange$: Observable<any>;
   searchresults$: Observable<any>;
+  analyticsStatus: string = "Show Chart";
```
Comment (simsausaurabh) [inline src/app/statsbox/statsbox.component.ts]: avoid using double quotes.

File: src/app/statsbox/statsbox.component.ts [hunk 3]
```diff
@@ -134,6 +136,15 @@ export class StatsboxComponent implements OnInit {
       return;
     };
   }
+  BoxToggle() {
+    if (this.analyticsStatus === "Show Chart") {
```
Comment (simsausaurabh) [inline src/app/statsbox/statsbox.component.ts]: avoid double quotes in other places.

**Comment by simsausaurabh** [comment]
@praveenojha33 Please make the cursor to be pointer and there is a small issue:

screenshot from 2018-07-07 00-28-37

when the button is not opened, it displays to close it and vice-versa.

**Comment by praveenojha33** [comment]
@simsausaurabh I have made requested changes Please review again.

**Pull request merged**